config/locales/en.yml

```diff
@@ -75,6 +75,31 @@ en:
       status: Status
       withdraw_submission: Withdraw Application
       confirm_withdraw: Withdraw your application
+  root:
+    index:
+      title: "Connecting hearts and homes"
+      title_highlight: "worldwide"
+      description: "A free application for pet foster and adoption organizations who want to save time and manage their operations in one place."
+      prod_cuality_1: "Easy to use"
```

Comment:
are these abbreviations? I am not familiar.

Reply:
bit of Spanglish... my bad. Just added a better name for this.